check-ostree.yaml

```diff
@@ -213,6 +213,27 @@
       when:
         - "'/dev/mapper/rootvg-rootlv' in result_sysroot_source.stdout"
         - fdo_credential == "true"
+        - os_name == "redhat"
```

Comment:
could you please use ansible_facts['distribution'] for different distros.

Reply:
Fixed.